lib/elasticsearchHandler.js

```diff
@@ -1,50 +1,133 @@
 "use strict";
+var ElasticsearchStore = module.exports = function ElasticsearchStore() { };
+
 var async = require("async");
+var debug = require("debug")("jsonApi:store:elasticsearch");
 var elasticsearch = require("elasticsearch");
-
-var ElasticsearchStore = module.exports = function ElasticsearchStore() {
+var _ = {
+  assign: require("lodash.assign")
 };
 
+
 ElasticsearchStore.prototype.ready = false;
-elasticsearch.resourcesToInitialise = [];
 
 ElasticsearchStore.prototype._buildQuery = function(request) {
```

Comment:
It's recommended that the query string query is not used in production and would be quite easy to convert this to use the proper query dsl.

I also understand that we're recommending that the user of this module overrides the search function for a more performant search, so I'm happy for this to be left as it is for now.